### PR TITLE
Add storage BatchCleanupRequest

### PR DESCRIPTION
**Pull request by ruhan1**
@sswguo this is for batch clean up.

### CODE REVIEW

File: src/main/java/org/commonjava/event/storage/BatchCleanupRequest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import java.util.Set;
+
+/**
+ * Cleanup multiple paths from multiple filesystems.
+ */
+public class BatchCleanupRequest
+{
+    private Set<String> paths;
+
+    private Set<String> filesystems;
```
Comment (sswguo) [inline src/main/java/org/commonjava/event/storage/BatchCleanupRequest.java:12]: So is this many-to-many ?

Comment (ruhan1) [reply src/main/java/org/commonjava/event/storage/BatchCleanupRequest.java:12]: Yes.

Comment (ruhan1) [reply src/main/java/org/commonjava/event/storage/BatchCleanupRequest.java:12]: I use two for loops, like for{ for{} }, to delete files.

Comment (sswguo) [reply src/main/java/org/commonjava/event/storage/BatchCleanupRequest.java:12]: I see. thanks.

**Comment by sswguo** [comment]
LGTM.

**Pull request merged**